x-pack/plugins/apm/typings/Error.ts

```diff
@@ -16,6 +16,7 @@ export interface Error extends APMDocV1 {
       pid: number;
     };
     service: ContextService;
+    [key: string]: any;
```

Comment:
added this to accommodate:
```
        <PropertiesTable
          propData={errorGroup.data.error.context[currentTab]}
          propKey={currentTab}
          agentName={agentName}
        />
```
We should follow up with more type definition on the possible properties of error.context type objects